Rubberduck.Parsing/Annotations/Concrete/IgnoreAnnotation.cs

```diff
@@ -1,8 +1,35 @@
 ﻿namespace Rubberduck.Parsing.Annotations
 {
     /// <summary>
-    /// Used for ignoring specific inspection results from a specified set of inspections.
+    /// @Ignore annotation, used for ignoring inspection results at member and local level.
     /// </summary>
+    /// <parameter name="Inspections" type="ParamArray (Identifier)">
+    /// This annotation optionally takes a comma-separated list of inspection names as argument. If no specific inspection is provided, then all inspections should ignore the annotated target.
+    /// </parameter>
+    /// <remarks>
+    /// Use the @IgnoreModule annotation to annotate at module level.
+    /// </remarks>
+    /// <example>
+    /// <module name="Class1" type="Class Module">
+    /// <![CDATA[
+    /// Option Explicit
+    /// Private InternalState As VBA.Collection
+    ///
+    /// '@Ignore
```

Comment:
This is not legal atm. Only `IgnoreModule` can be used without parameters.

Reply:
Nice catch! Will edit.